frontend/src/modules/feed/components/UserDropdown/UserDropdown.js

```diff
@@ -39,18 +36,18 @@ class DropdownRemote extends Component {
     this.setState({ isFetching: true });
     console.log(searchQuery);
 
-    axios.get(settings.API_USER_SEARCH).then(resp => {
-      const options = [];
-      resp.data.forEach(item => {
-        options.push({
-          text: `${item.first_name} ${item.last_name}`,
-          value: item.id,
-          image: { avatar: true, src: item.avatar_url },
-        });
-      });
+    // axios.get(settings.API_USER_SEARCH).then(resp => {
```

Comment:
More comments

Reply:
Temporary non functional code until the features are implemented in GraphQL.